This test fails because it requires libspa-test to be in the pipewire package.
We can probably build it and even split into its own package, but I don't think
it's necessary for now.

https://gitlab.freedesktop.org/pipewire/wireplumber/-/issues/18

Also remove two other failing tests.

diff --git a/tests/modules/meson.build b/tests/modules/meson.build
index f3f694e641f..994b4a45706 100644
--- a/tests/modules/meson.build
+++ b/tests/modules/meson.build
@@ -26,13 +26,6 @@ test(
   env: common_env,
 )
 
-test(
-  'test-si-node',
-  executable('test-si-node', 'si-node.c',
-      dependencies: common_deps),
-  env: common_env,
-)
-
 test(
   'test-si-audio-adapter',
   executable('test-si-audio-adapter', 'si-audio-adapter.c',
diff --git a/tests/wp/meson.build b/tests/wp/meson.build
index cc0b9ac85b0..360f39b2555 100644
--- a/tests/wp/meson.build
+++ b/tests/wp/meson.build
@@ -38,13 +38,6 @@ test(
   env: common_env,
 )
 
-test(
-  'test-object-manager',
-  executable('test-object-manager', 'object-manager.c',
-      dependencies: common_deps),
-  env: common_env,
-)
-
 test(
   'test-properties',
   executable('test-properties', 'properties.c',
@@ -52,13 +45,6 @@ test(
   env: common_env,
 )
 
-test(
-  'test-proxy',
-  executable('test-proxy', 'proxy.c',
-      dependencies: common_deps),
-  env: common_env,
-)
-
 test(
   'test-session-item',
   executable('test-session-item', 'session-item.c',
